hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/scm/protocolPB/StorageContainerLocationProtocolClientSideTranslatorPB.java

```diff
@@ -712,6 +717,67 @@ public boolean getReplicationManagerStatus() throws IOException {
 
   }
 
+  @Override
+  public boolean startContainerBalancer(double threshold, int idleiterations,
+         int maxDatanodesToBalance, long maxSizeToMove) throws IOException {
+    Preconditions.checkState(threshold >= 0,
```

Comment:
NIT. We can also add check threshold < 1.0.
```suggestion
    Preconditions.checkState(threshold >= 0.0D && threshold < 1.0D,
```